src/views/data/ControlBar.js

```diff
@@ -1,89 +1,79 @@
 import React from 'react';
-import {connect} from 'react-redux';
-import {bindActionCreators} from 'redux';
-import _ from 'lodash';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
 import timeout from 'util/timeout';
+import _ from 'lodash';
```

Comment:
Just curious why you rearranged the imports this way?

Reply:
honestly, I don't know but Im keeping it